identity_account/Cargo.toml

```diff
@@ -21,6 +21,7 @@ log = { version = "0.4", default-features = false }
 paste = { version = "1.0" }
 rand = { version = "0.8", default-features = false, features = ["std", "std_rng"] }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+serde_json = { version = "1.0", default-features = false }
```

Comment:
I don't think `serde_json` is explicitly necessary, we have the `ToJson` convenience trait from `identity_core` that exposes its functions.